### DISCUSSION TITLE
Newbie having trouble with the CSS selector syntax

### DISCUSSION THREAD

**Discussion by noodlejetski** [FreshRSS/FreshRSS#7130] [Q&A] [answered]
hello,

I'm trying to set up the CSS selector thing in order to get full articles in my feed, and since I have zero prior experience with CSS, it turns out to be a bit of a hurdle. I've managed to set it up with a bunch of feeds a while ago, but it appears that some websites have recently changed something on their end and I have to do it all over again.

here's the feed I'm having trouble with:  https://www.gamingonlinux.com/article_rss.php

from what I can tell by inspecting the article's source, the part I want to get grabbed is in the following section: 

`<div class="content group e-content"></div>`

which means that I should put that part in the CSS selector field in the feed settings. however, I've tried both `.content` and `.content.group.e-content` (which I _think_ is the proper way to format it for this purpose? again, zero CSS knowledge), but testing the preview results in the "The selector didn’t match anything" message. I've had some success by putting `article` instead, but that also grabs the footer, the social media icons (which render as HUGE in the feed), article tags and so on, and likewise I've had no success using CSS selector to remove those.

is there anything I should check/change/look into? any input is more than welcome, I'm using a FreshRSS instance managed by someone else on https://rss.hostux.net/, and it's currently running the `1.25.0-dev` version.

**Comment by math-GH** [comment]
Could you please support the OPML file for that feed?

See feed config at the end of list:
![grafik](https://github.com/user-attachments/assets/27de2e65-164e-4ed2-9d05-6951b4e2f138)

**Comment by noodlejetski** [comment]
my bad! here it is.
[f_37 GamingOnLinux Latest Articles.zip](https://github.com/user-attachments/files/18222077/f_37.GamingOnLinux.Latest.Articles.zip)

**Reply by Frenzie** [reply]
> .content.group.e-content (which I think is the proper way to format it for this purpose? 

That's correct. It's probably easier as something like `article .content`.

**Reply by noodlejetski** [reply]
and yet neither of those seems to grab the article's content when I'm testing it.

**Reply by Frenzie** [reply]
Something else must be the matter then. Either CSS selector works for me.

**Reply by math-GH** [reply]
For me it works too.

Maybe this is the reason why it does not work for you:
The CSS selectors work only for the new articles.
If you want to fetch the older articles too then reload them with the `reload articles` button (clear the cache before could help too)

![grafik](https://github.com/user-attachments/assets/322c9c77-e6b1-4401-8cb0-d5981b569efd)

**Reply by Alkarex** [reply]
Remember to use the preview button to test: 
![image](https://github.com/user-attachments/assets/d641b314-188e-4107-b0f4-116ca043d029)

**Comment by noodlejetski** [comment, marked as answer]
it looks like it's been an issue with this particular version of FreshRSS. the owner has recently updated it and full articles are being fetched properly according to CSS selector rules again.
